libconcurrent/includes/uthreads.h

```diff
@@ -27,4 +27,8 @@ int synchSpawnFiber(void *(*func)(void *), long arg);
 /// @brief Wait for all of the fibers that are running inside the current posix thread to finish their execution.
 void synchWaitForAllFibers(void);
 
+/// @brief This function returns an index of the running fiber in the current posix thread, i.e.
+/// an integer number between 0 and N-1, where N is the number of active fibers in the cirrent posix thread.
```

Comment:
`in the cirrent posix` --> `in the current posix`

Reply:
Fixed